docs/source/_static/main.css

```
@@ -0,0 +1,3 @@
+col {
```

Maybe this in case there are other kinds of tables elsewhere:
```suggestion
.longtable col {
```